cli_tools/diagnostics/main_windows.go

```diff
@@ -168,24 +167,48 @@ func gatherProgramLogs(logs chan logFolder, errs chan error) {
 	logs <- logFolder{"Program", runAll(commands, errs)}
 }
 
-func gatherEventLogs(logs chan logFolder, errs chan error) {
-	dirs := []string{`C:\Windows\System32\winevt\Logs`}
-	paths := make([]string, 0)
-	// Recursively gather all the log files in a directory
-	for len(dirs) > 0 {
-		dir := dirs[0]
-		dirs = dirs[1:]
-		files, _ := ioutil.ReadDir(dir)
-		for _, f := range files {
-			path := filepath.Join(dir, f.Name())
-			if f.IsDir() {
-				dirs = append(dirs, path)
-			} else {
-				paths = append(paths, path)
+func collectFilePaths(roots []string) ([]string, []error) {
```

Comment:
Please describe the method and its arguments and return value. Same for other methods you've added or modified.